postgres/rdspostgres/rdspostgres.go

```diff
@@ -72,15 +74,17 @@ func init() {
 // OpenPostgresURL opens a new RDS database connection wrapped with OpenCensus instrumentation.
 func (uo *URLOpener) OpenPostgresURL(ctx context.Context, u *url.URL) (*sql.DB, error) {
 	cf := new(rds.CertFetcher)
-	vals := u.Query()
```

Comment:
I am not sure if there was a point to this?

Reply:
There was not, and the `u.RawQuery` mutation violates the contract for the `URLOpener` interface. I caught this in at least one PR, but it must have slipped into the codebase. Sorry.